UnitTests/Tests/TRWeatherUpdateCacheSpec.m

```diff
@@ -29,6 +29,14 @@ - (NSURL *)latestWeatherUpdateURL;
     [[NSFileManager defaultManager] removeItemAtURL:weatherUpdateURLForTesting() error:nil];
 };
 
+NSDictionary* (^weatherConditionsWithTemperature) (NSNumber*) = ^NSDictionary* (NSNumber *temp) {
```

Comment:
Gotcha. I'd rather that we maintain a clean code base wherever possible. That means, in this case, keeping the existing code style for new code even if it's going to be replaced soon.